lib/experimental/results.nim

```diff
@@ -200,8 +200,8 @@ func `$`*[E](self: Result[void, E]): string =
   if self.isOk: "Ok()"
   else: "Err(" & $self.e & ")"
 
-template value*[T, E](self: Result[T, E]): T = get self
-template value*[T, E](self: var Result[T, E]): T = get self
+template value*[T, E](self: Result[T, E]): untyped = get self
```

Comment:
can you leave a comment here to indicate that this should become typed once the bug is fixed?

Reply:
With typed, do you mean `lent T` and `var T`? Or just `T` as was previously?

Reply:
I meant `T`